pkg/graph/resolvers/plan_basics.go

```diff
@@ -12,14 +16,66 @@ import (
 	"github.com/cmsgov/mint-app/pkg/storage/loaders"
 )
 
+func getFieldNameMap() map[string]string {
+	fieldNames := map[string]string{
+		"completeICIP":            "Complete ICIP",
+		"clearanceStarts":         "Clearance",
+		"clearanceEnds":           "Clearance",
+		"announced":               "Announce model",
+		"applicationsStart":       "Application period",
+		"applicationsEnd":         "Application period",
+		"performancePeriodStarts": "Performance period",
+		"performancePeriodEnds":   "Performance period",
+		"wrapUpEnds":              "Model wrap-up end date",
```

Comment:
This feels similar / duplicative to what's in getDateFieldsToCheck. Would it be possible to use this map for both purposes? ([You can iterate over the keys of a map](https://goplay.tools/snippet/TlQMdVWPHA6), which might mean this function could be used to get the list of field names to check as well)